src/Formatter/JsonStreamFormatter.php

```diff
@@ -43,6 +43,15 @@ static function (FormatterStream $runningValue, CollectionItem $item) use ($seri
         return $runningValue;
     }
 
+    /**
+     * @param FormatterStream $runningValue
+     */
+    public function serializeString(mixed $runningValue, Field $field, ?string $next): mixed
+    {
+        $runningValue->printf('"%s"', is_string($next) ? str_replace('"', '\"', $next) : null);
```

Comment:
Would it make more sense to use json_encode() to serialize the string, so that we escape things like \, as well?  Or would that cause double-encoding?

cf: https://3v4l.org/NfBe9

Reply:
I think you're right and I'll update the PR.